docs: reviewed markdown files (#50)
intershop/icm-migration-support: README.md

Reviewer: Please check the 3 new sections in Manual Migration Steps

```
gradlew migration:migrateAll -Ptarget=<path_to_7_10_project> -Psteps=<path_to_migration_steps> [-PnoAutoCommit]
```

### Migration Step by Step

Use the following commands to execute specific migration steps:

```
gradlew migration:migrateOne -Ptask=project -Ptarget=<path_to_7_10_project>/your_cartridge -Psteps=<path_to_single_migration_step> [-PnoAutoCommit]
gradlew migration:migrateOne -Ptask=projects -Ptarget=<path_to_7_10_project> -Psteps=<path_to_single_migration_step> [-PnoAutoCommit]
```

### Available Migration Steps
The migration steps are located in the `migration/src/main/resources/migration` folder of the project and are organized by major ICM versions. This allows to migrate from one major version to another while considering different starting points and supporting migration in more maintainable steps.

* [Migration 7.10 to 11](docs/migration-7.10-11.md)
* [Migration 11 to 12](docs/migration-11-12.md)

### Third Party Libraries
This project reuses code from the project [GradleKotlinConverter](https://github.com/bernaferrari/GradleKotlinConverter), licensed under the Apache License 2.0. 
The code from the project was adapted to fit the needs of this project. 

> All changes are marked in the "Gradle Kotlin DSL converter" code with a comment in the function `applyConversions()`.

Author: ```suggestion
Comments in the function `applyConversions()` of the _Gradle Kotlin DSL converter_ code identify all changes.
```
Is this what is meant here? @fgiesske-intershop
